core/src/main/res/drawable/core_ic_reload.xml

```diff
@@ -0,0 +1,24 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
1- can we use white color as a strokeColor to make it more generic?
2- requires auto-format